README.md

```diff
@@ -162,6 +162,32 @@ render() {
 }
 ```
 
+### How to use instanceRef with React Hooks?
+
+#### Import useRef 
+``` js
+import { useRef } from React;
+
+const instanceRef = useRef(null); 
+
+<EditorJs tools={EDITOR_JS_TOOLS} instanceRef={(instance) => (instanceRef.current = instance)} />
+```
+
+#### Save Editor Data
+```js
+let data = await instanceRef.current.save();
+```
+
+#### Clear Editor
+```js 
+instanceRef.current.clear();
+```
+
+#### Render Data inside the Editor
+```js
+instanceRef.current.render({ blocks }) 
+```
```

Comment:
```suggestion
```

Please delete this block or use enableReInitialize prop. I recommend delete block